polymetis/polymetis/python/polymetis/robot_interface.py

```diff
@@ -744,3 +784,32 @@ def move_ee_xyz(
             "The method 'move_ee_xyz' is deprecated, use 'move_to_ee_pose' with 'delta=True' instead."
         )
         return self.move_to_ee_pose(position=displacement, delta=True, **kwargs)
+
+    # MIRROR METHODS
+
+    def sync_with_mirror(self):
+        assert self.use_mirror_sim, "Mirror sim must be instantiated!"
+        assert not self.mirror_sim_robot, "Clean mirror after forward in order to sync!"
+        self.mirror_sim_client.sync(self)
+
+    def unsync_with_mirror(self):
+        assert self.use_mirror_sim, "Mirror sim must be instantiated!"
+        self.mirror_sim_client.unsync()
+
+    def setup_mirror_for_forward(self):
+        assert self.use_mirror_sim, "Mirror sim must be instantiated!"
+        assert (
+            not self.mirror_sim_client._state_setter
+        ), "Unsync before setting up for forward!"
+        self.mirror_sim_client.run(time_horizon=1)
+        self.mirror_sim_client.run_no_wait()
+        self.mirror_sim_robot = RobotInterface(
+            ip_address=self.mirror_ip,
+            port=self.mirror_port,
+            mirror_metadata=self.mirror_sim_client.metadata.get_proto(),
+        )
```

Comment:
Would it be easier to simply perform this by default in `__init__` if `use_mirror_sim == True`?